File: program/program/calculate-simple-interest/calculate_simple_interest.go
package main

import "fmt"

func main() {
	var amount, time, rate int
	fmt.Scanf("%d", &amount)
	fmt.Scanf("%d", &time)
	fmt.Scanf("%d", &rate)
	simpleInterest := (amount * time * rate) / 100
	fmt.Println(simpleInterest)
}

File: program/program/calculate-simple-interest/CalculateSimpleInterest.java
import java.util.Scanner;

public class CalculateSimpleInterest {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter");
        System.out.print("Principle Amount: ");
        double p = sc.nextDouble();
        System.out.print("Rate of Interest: ");
        double r = sc.nextDouble();
        System.out.print("Time(in Days): ");
        double t = sc.nextDouble();
        System.out.println("Simple Interest: " +simpleInterest(p,r,t));
    }
    static double simpleInterest(double principleAmount, double rate, double time){
        return (principleAmount * rate * time)/100;
    }
}
